parm/stage/master_gefs_RT.yaml.j2

```
@@ -0,0 +1,90 @@
+###################################################################
+# This is the master yaml for the GEFS
+#
+# Cycle, member, and RUN settings are set before including each
+# component yaml based on DO switches
+#
+# The included yamls are intended to be of the following structure:
+# key1:
+#     mkdir:
+#         - "COM directory to create"
+#     copy:
+#         - ["source_file", "destination_file"]
+#     link:
+#         - ["target", "link_name"]
+# key2:
+#     mkdir:
+#         - "COM directory to create"
+#     copy:
+#         - ["source_file", "destination_file"]
+#     link:
+#         - ["target", "link_name"]
+#
+# Any number of keys with nested mkdir, copy, and link are permitted
+# Jinja is permitted in this yaml, as long as the keys are:
+# - COMOUT_
+# - DO_ATM, DO_OCN, DO_ICE, etc.
+# For a full list see scripts/exglobal_stage_ic.py
+###################################################################
+
+
+# Set cycle variables
+# ------------------------
+{% set half_window = assim_freq // 2 %}
+{% set half_window_begin = (-half_window | string + "H") | to_timedelta %}
+{% set half_window_end = (half_window | string + "H") | to_timedelta %}
+{% if DOIAU and MODE == "cycled" %}
+    {% set model_start_date_current_cycle = current_cycle | add_to_datetime(half_window_begin) %}
+{% else %}
+    {% if REPLAY_ICS %}
+        {% set model_start_date_current_cycle = current_cycle | add_to_datetime(half_window_end) %}
+    {% else %}
+        {% set model_start_date_current_cycle = current_cycle %}
+    {% endif %}
+{% endif %}
```

```suggestion
{% set model_start_date_current_cycle = current_cycle | add_to_datetime(half_window_begin) %}
```
Based on the above comment, in RT, GFS will be running w/ IAU and in cycled mode.  There is no replay possible for running in RT